lib/tty/pager/system.rb

```diff
@@ -104,7 +104,7 @@ def self.executables
       #
       # @api private
       def self.command_exists?(command)
-        TTY::Which.exist?(command)
+        TTY::Which.exist?(command.split.first)
```

Comment:
Ideally the command argument wouldn't be changed in this place. This method is a simple wrapper around which `exist?` call. I would probably move it up to method that calls `command_exists?`